### ISSUE TITLE
Regex match in train_val_split_random()

### ISSUE THREAD

**Issue by jmontalvo94** [rte-france/Grid2Op#172]
## Environment
 - Grid2op version: `1.4.0`
 - System: WSL Ubuntu 20.04.2 LTS

## Bug description
When using `train_val_split_random()` an error is raised with any input.

## How to reproduce
```python
import grid2op
env = grid2op.make("l2rpn_case14_sandbox")
nm_env_train, nm_env_val = env.train_val_split_random()
```

## Current output
Regex matches any input, so it fails given the condition statement.
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jorge/code/bdonnot/Grid2Op/grid2op/Environment/Environment.py", line 1081, in train_val_split_random
    raise EnvError("The suffixes you can use for training data (add_for_train) "
grid2op.Exceptions.EnvExceptions.EnvError: Grid2OpException EnvError "The suffixes you can use for training data (add_for_train) should match the regex "^[a-zA-Z0-9]*$""
```

## Expected output
Splitted dataset.

**Comment by BDonnot** [comment]
Hello,

Thanks for this issue. Really well explained and can be reproduced easily !

I added the fix you proposed but committed myself because i also modified other part of the code.

I'll close the issue when new version will be out :-)

Benjamin

**Comment by BDonnot** [comment]
Fixed in release 1.5.0
